src/config/inter/addVault/add-oracle.js

```diff
@@ -22,7 +22,7 @@ const getManifestCall = harden([
       maxSubmissionCount: 1000,
       maxSubmissionValue:
         115792089237316195423570985008687907853269984665640564039457584007913129639936n,
-      minSubmissionCount: 3,
+      minSubmissionCount: 3, // TODO: parameterize
```

Comment:
@otoole-brendan this means that `minSubmissionCount` will be 3 on xnet as well. Please confirm that this is OK.

Reply:
Hmm that would mean needing to use 3x gov accounts with oracle rights to change price right? How are we handling this on Emerynet where control oracles with 2x Gov accounts? 
I'm not keen on messing with the way we change prices on Emerynet (and Xnet) - that both require Gov1 and Gov 2 alone to do so unless we deem it necessary

Reply:
@otoole-brendan writes elsewhere:
> Let’s keep it as a constant across all networks then on the basis we don’t want to do more work by updating the app